packages/core/package.json

```diff
@@ -31,6 +47,3926 @@
         "./notifications": "./dist/src/notifications/index.js"
     },
     "contributes": {
+        "configuration": {
```

Comment:
this doesn't look right. this stuff now lives in `packages/toolkit/package.json` , not the core.

Reply:
corrected this, thank you for catching!